packages/gatsby/src/utils/webpack-hmr-hooks-patch.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+
+// @ts-ignore: This file assume run at browsers.
+const originalFetch = global.fetch
+// @ts-ignore
+delete global.fetch
```

Comment:
can you try something like this? https://github.com/eslint/eslint/pull/763

Reply:
not sure if this helps. i think these are fine for this file

Reply:
I tried it, but the reasons of CI fail in 81a9274 and d49470a are the same.
The error message `Error: You must provide the URL of lib/mappings.wasm by calling SourceMapConsu  mer.initialize({ 'lib/mappings.wasm': ... }) before using SourceMapConsumer` seems caused by https://github.com/gatsbyjs/gatsby/pull/22280#issuecomment-599228141.